src/minecraftServers/Controllers/ServersController.cs

```diff
@@ -42,30 +42,27 @@ public ActionResult<Server> Add(ServerDto server)
         var receivedServer = _serversRep.GetByIp(server.Ip);
         if (receivedServer != null)
         {
-            return BadRequest("This server already exist !");
+            return BadRequest("This server is already exist !");
         }
         var updatedServer = new Server(server.Ip, 0, server.Name, 0);
         return Ok(_serversRep.Add(updatedServer));
     }
 
     [HttpDelete("{id}")]
-    public ActionResult Delete(uint id)
+    public ActionResult Delete(int id)
     {
         _serversRep.Delete(id);
         return Ok();
     }
 
     [HttpPut("{id}")]
-    public ActionResult Update(ServerDto server, uint id)
+    public ActionResult Update(ServerDto server, int id)
     {
-        var exsistedServer = _serversRep.GetById(id);
-        if (exsistedServer == null)
+        var isFound = _serversRep.Update(server,id);
```

Comment:
The `is` prefix is usually used for `bool` variables, I think, it'll be more clear if you change variable's name